test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -262,25 +244,152 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
           hello-world-svc (get-service app :HelloWorldService)]
       (is (= (hello-world hello-world-svc) "hello world")))))
 
+(deftest duplicate-service-definitions
+  (testing "Duplicate service definitions causes error with filename and line numbers"
```

Comment:
Would it make sense to have a test of what happens if there are duplicate service definitions between two different files? Not sure that with the code paths that that would be testing anything different, but it is a scenario that I find myself wondering about as I review this PR.